### PR TITLE
helm: Add hubble section

### PR DESCRIPTION
**Pull request by michi-covalent**
2 commits:

- add a hidden `cilium observe` subcommand to get flows from hubble.
- update helm.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/10358)

### CODE REVIEW

**Review by aanm — CHANGES_REQUESTED** [review]
I don't see where `HUBBLE_DEFAULT_SOCKET_PATH` nor `HUBBLE_GROUP_NAME` are used in the cilium's code base so I assume this is waiting for #10238 to be merged first?

**Comment by michi-covalent** [comment]
> I don't see where `HUBBLE_DEFAULT_SOCKET_PATH` nor `HUBBLE_GROUP_NAME` are used in the cilium's code base so I assume this is waiting for #10238 to be merged first?

right. these are env variables in hubble that will be available once #10238 is merged, although this pr is not really blocked on it since it doesn't add any hubble-related settings by default unless you specify them in helm command.

**Comment by michi-covalent** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

File: install/kubernetes/cilium/values.yaml [hunk 1]
```diff
@@ -359,3 +359,42 @@ global:
...
+    #   - "unix:///var/run/cilium/hubble.sock"
+    #
+    # You can specify the list of metrics from the helm CLI:
+    #
+    #   --set global.hubble.listenAddresses={unix:///var/run/cilium}
+    #
+    # Hubble is disabled if the list is empty.
+    listenAddresses: []
+    # Buffer size of the channel Hubble uses to receive monitor events. Defaults to 128.
+    eventQueueSize: ~
```
Comment (gandro) [inline install/kubernetes/cilium/values.yaml:377]: Apparently I missed this in the other PR, but 128 (which is taken from stand-alone) is way to too small for embedded mode, Hubble will drop most flows with this setting (https://github.com/cilium/hubble/issues/161).

This means we need to the bump the default for `option.HubbleEventQueueSize` here as well:
https://github.com/cilium/cilium/blob/8d28ab13be7d2aeca75702b89445a0d26c4815ca/daemon/daemon_main.go#L738

I propose we use the same value as we use for `MonitorQueueSize`:
https://github.com/cilium/cilium/blob/8d28ab13be7d2aeca75702b89445a0d26c4815ca/pkg/option/config.go#L2166-L2171

**Review by gandro — COMMENTED** [review]
See comment regarding event queue size. I'm fine if we fix this in a follow-up PR though.

File: install/kubernetes/cilium/values.yaml (hunk 1 above)
Comment (tgraf) [reply install/kubernetes/cilium/values.yaml:377]: Let's fix it in a separate PR so it's documented properly why we bumped it.

**Review by tgraf — APPROVED** [review]

File: install/kubernetes/cilium/values.yaml (hunk 1 above)
Comment (michi-covalent) [reply install/kubernetes/cilium/values.yaml:377]: https://github.com/cilium/cilium/pull/10488 cc @gandro

**Pull request merged**